rdr_client/hpo_assigner.py

```diff
@@ -64,10 +65,10 @@ def main(client):
       if hpo == 'UNSET':
         participant['providerLink'] = []
       else:
-        participant['providerLink'] = [{'primary': True,
```

Comment:
_ORG_PREFIX is now unused and can be removed?

Reply:
yup